microcosm_flask/operations.py

```diff
@@ -27,6 +29,7 @@ class Operation(Enum):
     """
     # discovery operation
     Discover = OperationInfo("discover", "GET", NODE_PATTERN, 200)
+    DiscoverVersion = OperationInfo("discover_version", "GET", VERSIONED_NODE_PATTERN, 200)
```

Comment:
Swagger now embeds the version into its endpoint pattern so we can have multiple at once with different versions.